packages/flutter_tools/lib/src/macos/xcode_validator.dart

```diff
@@ -36,6 +36,9 @@ class XcodeValidator extends DoctorValidator {
         if (xcodeVersionInfo.contains(',')) {
           xcodeVersionInfo = xcodeVersionInfo.substring(0, xcodeVersionInfo.indexOf(','));
         }
+        if (_xcode.buildVersion != null) {
+          messages.add(ValidationMessage('Build ${_xcode.buildVersion}'));
+        }
       }
```

Comment:
As you have it `buildVersion` won't be null if `versionText` is null, but pull it out of the if statement anyway in case that ever changes and they aren't set dependently.
```suggestion
      }
      if (_xcode.buildVersion != null) {
        messages.add(ValidationMessage('Build ${_xcode.buildVersion}'));
      }
```